docs/operators/udp_input.md

```diff
@@ -4,27 +4,56 @@ The `udp_input` operator listens for logs from UDP packets.
 
 ### Configuration Fields
 
-| Field             | Default          | Description                                                                                        |
-| ---               | ---              | ---                                                                                                |
-| `id`              | `udp_input`      | A unique identifier for the operator                                                               |
-| `output`          | Next in pipeline | The connected operator(s) that will receive all outbound entries                                   |
-| `listen_address`  | required         | A listen address of the form `<ip>:<port>`                                                         |
-| `write_to`        | $                | The body [field](/docs/types/field.md) written to when creating a new log entry                    |
-| `attributes`      | {}               | A map of `key: value` pairs to add to the entry's attributes                                       |
-| `resource`        | {}               | A map of `key: value` pairs to add to the entry's resource                                         |
-| `add_attributes`  | false            | Adds `net.transport`, `net.peer.ip`, `net.peer.port`, `net.host.ip` and `net.host.port` attributes |
+| Field             | Default          | Description                                                                                                        |
+| ---               | ---              | ---                                                                                                                |
+| `id`              | `udp_input`      | A unique identifier for the operator                                                                               |
+| `output`          | Next in pipeline | The connected operator(s) that will receive all outbound entries                                                   |
+| `listen_address`  | required         | A listen address of the form `<ip>:<port>`                                                                         |
+| `write_to`        | $                | The body [field](/docs/types/field.md) written to when creating a new log entry                                    |
+| `attributes`      | {}               | A map of `key: value` pairs to add to the entry's attributes                                                       |
+| `resource`        | {}               | A map of `key: value` pairs to add to the entry's resource                                                         |
+| `add_attributes`  | false            | Adds `net.transport`, `net.peer.ip`, `net.peer.port`, `net.host.ip` and `net.host.port` attributes                 |
+| `max_log_size`    | `1MiB`           | The maximum size of a log entry to read before failing. Protects against reading large amounts of data into memory |
```

Comment:
Since UDP sets a maximum size, and since we are defaulting to that size, I think parameter is only serving as a memory tuning knob. Is a worthwhile use case? Is there another use case I'm missing?

If we do wish to keep it, then we need to reflect the correct default value, and change the description to explain what it can be used for.

Reply:
not sure. Does max_log_size for other operators have other purpose than memory tune?

Reply:
I think that in the contexts of `tcp_input` and `file_input`, this setting works as described, in that it can "Protect against reading large amounts of data into memory". 

In any case, I see that you've removed it from the doc, but it is still present throughout the code. We can remove related values such as `minMaxLogSize` as well, since they only served to validate this setting.

Reply:
I think that in the contexts of `tcp_input` and `file_input`, this setting works as described, in that it can "Protect against reading large amounts of data into memory". 

In any case, I see that you've removed it from the udp.